pom.xml

```diff
@@ -93,6 +93,12 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>ch.qos.logback</groupId>
```

Comment:
Why the logback dependency?

Reply:
this was just to see the netty detailed logs doing testing. it is not really needed.